BlazarUI/app/scripts/models/Log.js

```diff
@@ -251,11 +247,12 @@ class Log extends Model {
         console.warn("We couldn't decode some UTF-8, so we're displaying this log line undecoded:\n", line);
       }
 
-      offsetRunningTotal += getByteLength(line);
+      // increment for next loop interation
+      offsetRunningTotal += getByteLength(line + NEW_LINE);
```

Comment:
This ensures that the offsets in the HTML element ids match the offsets sent to the api and that blank lines have unique element ids.